core-web/libs/sdk/angular/src/lib/layout/dotcms-layout/dotcms-layout.component.ts

```diff
@@ -35,10 +35,12 @@ import { RowComponent } from '../row/row.component';
 @Component({
     selector: 'dotcms-layout',
     standalone: true,
-    imports: [RowComponent],
+    imports: [RowComponent, AsyncPipe],
     template: `
-        @for (row of this.pageAsset?.layout?.body?.rows; track $index) {
-            <dotcms-row [row]="row" />
+        @if (pageAsset$ | async; as page) {
+            @for (row of this.page?.layout?.body?.rows; track $index) {
+                <dotcms-row [row]="row" />
+            }
```

Comment:
I wish we could use signals

Reply:
I would love that but we should support older Angular versions.

cc: @fmontes @zJaaal @KevinDavilaDotCMS